packages/@aws-cdk/aws-s3/lib/notifications-resource/notifications-resource-handler.ts

```diff
@@ -53,22 +57,12 @@ export class NotificationsResourceHandler extends Construct {
   /**
    * The role of the handler's lambda function.
    */
-  public readonly role: iam.Role;
+  public readonly role: iam.IRole;
```

Comment:
Technically, this is a breaking change. But, as far as I know, this is only used inside the CDK code, so we should be fine. If anyone has any call out about this, now is the time to speak up :-)

Reply:
@otaviomacedo You're right, this class isn't used outside CDK, and its actually not even exported from the `s3` package at all:

https://github.com/aws/aws-cdk/blob/534babde886646b6cd01f8799e3b4ed6db221263/packages/%40aws-cdk/aws-s3/lib/index.ts#L1-L8

So should not be considered a breaking change from `jsii` standpoint. I do some breaking changes errors in the logs, but they appear to be coming from somewhere else...